### PR TITLE
Frmw970 guides

### PR DESCRIPTION
**Pull request by cathysiller**
JIRA: https://jira.rax.io/browse/FRMW-755 and https://jira.rax.io/browse/FRMW-970

### LGTMs
- [x] Dev LGTM
- [x] Dev+Vidyo LGTM
- [x] Design LGTM

# Before
encoreui overview-encoreui-before

# After
encoreui overview-encoreui-after

### CODE REVIEW

File: demo/templates/overview.html [hunk 1]
```diff
@@ -8,38 +8,15 @@
...
-        <p>How to write and run tests</p>
-
-        <a href="#/guides/html-styleguide">HTML Styleguide</a>
-        <p>Guidelines and standards for writing HTML</p>
-
-        <a href="#/guides/css-styleguide">CSS Styleguide</a>
-        <p>Guidelines and standards for writing CSS and LESS</p>
-
-        <a href="#/guides/js-styleguide">JS Styleguide</a>
-        <p>Guidelines and standards for writing JS</p>
```
Comment (CITguy) [inline demo/templates/overview.html]: Demo app/documentation is targeted at Consumers, so these links (which have content targeted to Contributors) were removed to reduce unnecessary "noise".

File: demo/templates/overview.html [hunk 2]
```diff
@@ -78,7 +55,16 @@
         </a>
         <p>List of what features and updates are included in each release</p>
 
-        <a href="#/guides/common-questions">Common Questions</a>
+        <a href="https://github.com/rackerlabs/encore-ui/wiki/3.0-Migration-Guide">
+          <i class="fa fa-external-link"></i>
+          3.0 Migration Guide
+        </a>
+        <p>Instructions on upgrading from EncoreUI 2.x to 3.0</p>
```
Comment (CITguy) [inline demo/templates/overview.html]: New link to guide Consumers toward the migration guide.

File: guides/bower.md [hunk 3]
```diff
@@ -1,15 +0,0 @@
...
-
-Updates to this repo have been automated through the `grunt shipit` command. When `shipit` is run, it builds all relevant files, pushes to the public CDN, but also build the corresponding files for Bower and pushes a new version to this separate repo. You must have push privileges to run this.
-
-## Note
-
-Ideally, this would all be handled within the same repo and no separate push to bower would have to take place. This should be investigated further as time permits. A good starting point is [Angular UI Publisher](https://github.com/angular-ui/angular-ui-publisher).
-
-# Local Dev with Bower Link
-
-If you find yourself working on an Encore app and need to work with an EncoreUI component that hasn't been released in an official version, you can use `bower link` to link a local install of EncoreUI (with the latest/greatest) with your app's bower directory. For more information, read the [Live Development of a Bower Component](https://oncletom.io/2013/live-development-bower-component/) blog post.
```
Comment (CITguy) [inline guides/bower.md]: Now lives at https://github.com/rackerlabs/encore-ui/wiki/Publishing-Via-Bower

File: guides/common-questions.md [hunk 4]
```diff
@@ -1,32 +0,0 @@
...
-
-A breaking change is one that *requires* users of the changed component to modify their own code.
-
-### How are issues prioritized?
-
-As a small team with limited resources, Encore relies on prioritization to drive development. However, EncoreUI is open source and new components or functionality may be submitted in a pull request. Still, there are times when EncoreUI's team must review or tweak design and functionality of features before they are ready to be brought in. Please see our [Contribution Guide](../CONTRIBUTING.md) if you are planning on submitting a pull request.
-
-Those looking to have new features or designs prioritized by the team should attend the Encore Stakeholder Meeting, where developers, designers, contributors, and consumers of EncoreUI meet weekly to review recent releases, current work, and upcoming needs. This meeting is the best way to request support from EncoreUI for designs and features you need that haven't been developed yet.
-
-E-mail the [EncoreUI mailing list](mailto:encoreui@lists.rackspace.com) if you need an invitation to the meeting.
```
Comment (CITguy) [inline guides/common-questions.md]: Now lives at https://github.com/rackerlabs/encore-ui/wiki/Common-Questions

File: guides/css-styleguide.md [hunk 5]
```diff
@@ -1,151 +0,0 @@
...
-.rxCheckbox { ... }
-.rxSelect { ... }
-.rxRadio { ... }
-```
-
-For attribute directives that do not wrap their target element, use the attribute name selector.
-
-```css
-[rx-form] { ... }
-```
```
Comment (CITguy) [inline guides/css-styleguide.md]: Now lives at https://github.com/rackerlabs/encore-ui/wiki/CSS-Styleguide

File: guides/html-styleguide.md [hunk 6]
```diff
@@ -1,43 +0,0 @@
...
-    </ul>
-  </li>
-</ul>
-```
-
-### HTML Max Line Length
-The maximum line length for HTML files should be 120 characters. Review line
-lengths during coding of HTML files, and break to a new line if it exceeds
-120 characters. Hyperlinks are an exception to this rule. Hyperlinks can
-exceed the 120 character limit and must not be broken or separated.
```
Comment (CITguy) [inline guides/html-styleguide.md]: Now lives at https://github.com/rackerlabs/encore-ui/wiki/HTML-Styleguide

File: guides/js-styleguide.md [hunk 7]
```diff
@@ -1,127 +0,0 @@
...
-See [the jQuery Style Guide](http://contribute.jquery.org/style-guide/js/#type-checks) for guidance on checking data types of variables. For instances where using the `jQuery` object is recommended, instead use the matching [Lo-Dash](https://lodash.com) utility method:
-
- - [Plain Object](http://lodash.com/docs#isPlainObject): `_.isPlainObject(object)`
- - [Function](http://lodash.com/docs#isFunction): `_.isFunction( object )`
- - [Array](http://lodash.com/docs#isArray): `_.isArray( object )`
-
-## Resources
-
-* [Airbnb JavaScript Style Guide](https://github.com/airbnb/javascript)
-* [Google's JavaScript Style Guide](http://google-styleguide.googlecode.com/svn/trunk/javascriptguide.xml)
```
Comment (CITguy) [inline guides/js-styleguide.md]: Now lives at https://github.com/rackerlabs/encore-ui/wiki/Javascript-Styleguide

File: guides/testing.md [hunk 8]
```diff
@@ -1,255 +0,0 @@
...
-    });
-
-    it('interactive', function () {
-        myPage.myComponent.interactWith(); // trigger some click that expands something
-        // this will capture the component in its current "interacted" state
-        screenshot.snap(this, myPage.myComponent.rootElement);
-    });
-
-});
-```
```
Comment (CITguy) [inline guides/testing.md]: Now lives at https://github.com/rackerlabs/encore-ui/wiki/Testing

File: guides/ui-setup.md [hunk 9]
```diff
@@ -1,188 +0,0 @@
...
-
- 1. Go to `encore-ui/bower` in your EncoreUI directory
- 2. Run `bower link`. Behind the scenes, this will create a global symlink in `~/.local/share/bower/links/`
- 3. In your local application's directory, run `bower link encore-ui`. This will tell your application to grab EncoreUI from the local bower links, instead of from the official bower repositories. Restart your application.
- 4. When you are done testing your EncoreUI changes in your local project, run `bower update` from your project directory. This will tell it to start using the official bower repositories once again.
-
-Note that if you make further changes in your EncoreUI directory, you'll have to run `grunt copy:bower` from within the EncoreUI directory to cause these changes to become visible in the local symlink directory. Requiring this step is a failure on our part, and in the future we'll modify our EncoreUI `watch` task to make sure this happens automatically.
```
Comment (CITguy) [inline guides/ui-setup.md]: Now lives at https://github.com/rackerlabs/encore-ui/wiki/Developer-Setup

File: guides/using-encoreui.md [hunk 10]
```diff
@@ -1,44 +0,0 @@
...
-
-While the intention of EncoreUI is for it to be used in conjunction with AngularJS, it is possible to use just the styles (similar to how Twitter Bootstrap can be used without JS).
-
-The entirety of the EncoreUI CSS is available via the main CSS file. To use, include it on your page [as described above](#cdn).
-
-> Disclaimer: At any point in time, an HTML change to the Angular directive template can change the HTML needed for the CSS to work properly. Be sure when upgrading major/minor versions of the CSS file to check [the releases](https://github.com/rackerlabs/encore-ui/releases) for details on what has been updated.
-
-## Further Guidance
-
-For further help, reach out to the EncoreUI team via [our support information](https://github.com/rackerlabs/encore-ui#general-support).
```
Comment (CITguy) [inline guides/using-encoreui.md]: Now lives at https://github.com/rackerlabs/encore-ui/wiki/Using-EncoreUI

**Comment by ssalinas24** [comment]
Design LGTM

**Pull request merged**